libs/ui/src/button.test.tsx

```diff
@@ -111,22 +259,3 @@ describe('renders Button', () => {
     expect(onPress).toHaveBeenCalledTimes(3);
   });
 });
-
-test('disabling works for both touch events and click events', () => {
-  const onPress = jest.fn();
-  render(
-    <Button onPress={onPress} disabled>
-      Disabled Button
-    </Button>
-  );
-  const button = screen.getByText('Disabled Button');
-
-  // Click is disabled
-  userEvent.click(button);
-  expect(onPress).toHaveBeenCalledTimes(0);
-
-  // Touch is disabled
-  fireEvent.touchStart(button, createTouchStartEventProperties(100, 100));
-  fireEvent.touchEnd(button, createTouchEndEventProperties(100, 100));
-  expect(onPress).toHaveBeenCalledTimes(0);
-});
```

Comment:
Rationale for removing this test?

Reply:
Oh, it looks like it moved up into the other `disabled` test? Seems good. I remember when we got pinged on this bug so just want to make sure we have coverage.